util/EFSOI_Utilities/src/loadbal_efsoi.f90

```diff
@@ -0,0 +1,381 @@
+module loadbal_efsoi
```

Comment:
how different is `loadbal_efsoi` from `loadbal` in the EnKF.
Again we should link to the `enkf` library and not try to duplicate code.

Reply:
I'll revisit this

Reply:
The problem in part is that the native loadbal is expecting to work with statevec, which I don't immediately see how I can convince to do what I need (read in two forecasts and an analysis). I have an idea or two, which would involve copying the state vector to new variables, but that would involve a vigorous refactoring. I understand the maintenance concerns and general annoyance of duplicated code, but loadbal also doesn't seem to have drifted much since 2016.

Reply:
@aerorahul Any thoughts about this?